.github/workflows/verify_build.yaml

```diff
@@ -21,3 +21,12 @@ jobs:
           pull: true
           push: false
           tags: test-build
+
+      - name: rust-actions-runner-1.58
```

Comment:
Not sure we need a different runner, since we run `docker build .` on it anyway.